src/styles/index.scss

```diff
@@ -1,3 +1,35 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap");
+
+/* #region main-settings */
+
+html,
+body,
+div,
+a,
+section,
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
   margin: 0;
+  padding: 0;
 }
+
+/* #endregion */
+
+/* #region variables */
+
+$base-with: 200px;
+$main-with: $base-with - 34px;
+$blue-color: rgb(0, 172, 220);
+$white-color: rgb(255, 255, 255);
+$main-color: rgb(6, 11, 53);
+$second-color: rgb(97, 96, 112);
```

Comment:
Move it to variables file

Reply:
<img width="618" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/29507567-fd23-4ac0-86d0-5aa279b98201">
try to fix this comment